### PR TITLE
Postponed: Add Web target

### PR DESCRIPTION
**Pull request by michael-markl**

### CODE REVIEW

File: frontend/web/manifest.json [hunk 1]
```diff
@@ -0,0 +1,35 @@
+{
+    "name": "frontend",
+    "short_name": "frontend",
```
Comment (maxammann) [inline frontend/web/manifest.json:3]: Probably needs whitelabeling aswell. We could also just build this only for the Ehrenamtskarte Bayern app.

In both cases -> New issue about whitelabeling and do that in a separate issue.

Comment (maxammann) [reply frontend/web/manifest.json:3]:
```suggestion
    "name": "Ehrenamtskarte Bayern",
    "short_name": "Ehrenamt",
```

**Review by maxammann — COMMENTED** [review]
Looks good already! Thinks to do in this PR:

* Add a CI job so we know when things break
* Create a task for whitelabeling this
* Somehow integreate this into our build concept. E.g. we want to define a configuration for the web target for each build config. So that we can exclude the tabs for verification in the web version (of maybe not, if we want to allow verification :D).

**Comment by ztefanie** [comment]
@michael-markl What is your plan with this PR? Should we close it or should we celebrate it's first birthday soon :D :partying_face:

**Comment by maxammann** [comment]
I'm all for celebrating. Then merge on its birthday :P

**Comment by f1sh1918** [comment]
Hm shouldn't be that big issue to finish the task. Not sure how good the `SecureStorage` works for web but then we could also include verification.
@dkehne do you think this could be useful soonish?
Maybe i can make it till birthday :)

**Comment by michael-markl** [comment]
I think we should not ship verification in the web app.

Also, we should check again, if the Flutter web target is accessible at all.
If not, we might not be able (or we might consider not) to sell the web target.

**Comment by maxammann** [comment]
Hm actually that is a valid point. There is no secure storage in web browsers. So maybe it is actually risky to expose such a thing.


Let's just close?

**Comment by michael-markl** [comment]
I think the original idea was never to bring "Ausweisen" to the web target, but only the map and the search of accepting stores. That's still a valid use case for a web app.

**Comment by f1sh1918** [comment]
I think the biggest web target issue is lacking seo Support.
Even I think it's nice to just check stores on web but may not justify the effort

**Comment by maxammann** [comment]
Yeah I think we should invest the time to disable the Ausweisen tab. We probably don't want to use experimental stuff.

The map functionality would be fun though.

**Comment by ztefanie** [comment]
Tomorrow we can selebrate the second birthday of this PR. I will now close this, feel free to reopen it if you have a different opinion on it. We can still reopen this anytime we want to invest time in this feature.

**Pull request closed without merging**